engine/src/state.rs

```diff
@@ -58,14 +59,14 @@ impl State {
         let mut tournament = true;
         // Did white open with a Queen?
         if let Some((piece_str, _)) = history.moves.get(0) {
-            let piece = Piece::from_string(piece_str)?;
+            let piece = Piece::from_str(piece_str)?;
```

Comment:
if we wanted, we could now switch these calls to be
```suggestion
            let piece: Piece = piece_str.parse()?;
```